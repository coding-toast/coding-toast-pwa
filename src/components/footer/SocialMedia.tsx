import * as React from 'react'
import { Link } from 'gatsby'
import RedditIcon from '../../images/Reddit.svg'
import TwitterIcon from '../../images/Twitter.svg'
import RSSIcon from '../../images/RSS.svg'
import FooterStyles from './Footer.module.scss'

const SocialMedia: React.FC = () => {
  return (
    <div>
      <p className={FooterStyles.lead}>Follow Us</p>
      <div className={FooterStyles.row}>
        <div className={FooterStyles.column}>
          <Link to="/">
            <img src={RedditIcon} className={FooterStyles.icon} alt="Reddit" />
          </Link>
        </div>
        <div className={FooterStyles.column}>
          <Link to="/">
            <img src={TwitterIcon} className={FooterStyles.icon} alt="Twitter" />
          </Link>
        </div>
        <div className={FooterStyles.column}>
          <Link to="/">
            <img src={RSSIcon} className={FooterStyles.icon} alt="RSS" />
          </Link>
        </div>
      </div>
    </div>
  )
}

export default SocialMedia
