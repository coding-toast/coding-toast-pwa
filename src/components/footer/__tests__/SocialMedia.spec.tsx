import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SocialMedia from '../SocialMedia'

describe('<SocialMedia />', () => {
  it('renders correctly', () => {
    const component = shallow(<SocialMedia />)

    expect(toJson(component)).toMatchSnapshot()
  })
})
