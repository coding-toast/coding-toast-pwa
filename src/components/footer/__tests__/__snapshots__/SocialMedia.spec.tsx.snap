// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocialMedia /> renders correctly 1`] = `
<div>
  <p
    className="lead"
  >
    Follow Us
  </p>
  <div
    className="row"
  >
    <div
      className="column"
    >
      <mockConstructor
        to="/"
      >
        <img
          alt="Reddit"
          className="icon"
          src="test-file-stub"
        />
      </mockConstructor>
    </div>
    <div
      className="column"
    >
      <mockConstructor
        to="/"
      >
        <img
          alt="Twitter"
          className="icon"
          src="test-file-stub"
        />
      </mockConstructor>
    </div>
    <div
      className="column"
    >
      <mockConstructor
        to="/"
      >
        <img
          alt="RSS"
          className="icon"
          src="test-file-stub"
        />
      </mockConstructor>
    </div>
  </div>
</div>
`;
